docs/guide/workflow/workflow.md

```diff
@@ -76,6 +76,16 @@ The notification is displayed in the user menu:
 
 ![Notification about content to review](../img/workflow_notification.png)
 
+#### Draft locking
+
+You can enable draft locking, so that when a user sends a draft to review, only 
```

Comment:
Not sure if that is relevant here, but there is currently no way to disable the feature, it is always enabled.